FaceID/TestCode/camTest.py

```diff
@@ -0,0 +1,38 @@
+import cv2
+import numpy as np
+from PIL import Image
+import os
+
+
+# Path for face image database
+path = 'dataset'
+recognizer = cv2.face.LBPHFaceRecognizer_create()
+detector = cv2.CascadeClassifier("Cascades/haarcascade_frontalface_default.xml");
+
+# function to get the images and label data
+def getImagesAndLabels(path):
+
+    imagePaths = [os.path.join(path,f) for f in os.listdir(path)]     
+    faceSamples=[]
```

Comment:
Can you add spaces around the `=`. It makes it more readable and helps with consistency